letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -495,10 +504,18 @@ if [ "$1" = "--le-auto-phase2" ]; then
     say "Creating virtual environment..."
     DeterminePythonVersion
     rm -rf "$VENV_PATH"
-    if [ "$VERBOSE" = 1 ]; then
-      "$LE_PYTHON" -m venv "$VENV_PATH"
+    if [ "$PYVER" -le 27 ]; then
```

Comment:
Did you hang onto this branch on purpose?

Reply:
Yeah. For most systems we're still using Python 2.7 and `python -m venv` is [new in Python 3.3+](https://docs.python.org/3/library/venv.html).

Reply:
Ah, I had a brain fart. <=2.7, not <2.7. r+!